hedera-node/hedera-app-spi/src/testFixtures/java/com/hedera/node/app/spi/fixtures/info/FakeNetworkInfo.java

```diff
@@ -34,10 +34,29 @@ public class FakeNetworkInfo implements NetworkInfo {
 
     private static final List<NodeInfo> FAKE_NODE_INFOS = List.of(
             fakeInfoWith(
-                    false, 2L, "Alpha", AccountID.newBuilder().accountNum(3).build()),
-            fakeInfoWith(true, 4L, "Bravo", AccountID.newBuilder().accountNum(4).build()),
+                    2L,
+                    AccountID.newBuilder().accountNum(3).build(),
+                    30,
+                    "333.333.333.333",
+                    50233,
+                    "3333333333333333333333333333333333333333333333333333333333333333",
+                    "Alpha"),
             fakeInfoWith(
-                    false, 8L, "Charlie", AccountID.newBuilder().accountNum(5).build()));
+                    4L,
+                    AccountID.newBuilder().accountNum(4).build(),
+                    40,
+                    "444.444.444.444",
```

Comment:
Can we handle IPv6 address format?  Should we test that?

Reply:
I'm not sure TBH. From the services perspective, we don't do anything with this string, it is just passed to mirror node and platform.